src/OrbitGl/MemoryTrack.h

```diff
@@ -52,6 +52,19 @@ using SystemMemoryTrack = MemoryTrack<kSystemMemoryTrackDimension>;
 constexpr size_t kCGroupAndProcessMemoryTrackDimension = 4;
 using CGroupAndProcessMemoryTrack = MemoryTrack<kCGroupAndProcessMemoryTrackDimension>;
 
+const std::array<Color, kSystemMemoryTrackDimension> kSystemMemoryTrackColors{
+    Color(231, 68, 53, 255),  // red
+    Color(246, 196, 0, 255),  // orange
+    Color(87, 166, 74, 255)   // green
+};
+
+const std::array<Color, kCGroupAndProcessMemoryTrackDimension> kCGroupAndProcessMemoryTrackColors{
+    Color(231, 68, 53, 255),   // red
+    Color(249, 96, 111, 255),  // warm red
+    Color(246, 196, 0, 255),   // orange
+    Color(87, 166, 74, 255)    // green
```

Comment:
Where were these specific color values taken from? Or were they hand-picked? I'm asking because if there is a special "source" it would be useful to have a comment in case one then needs to add more or change them.

Reply:
Thanks for the suggestioin! I have added a comment to mention the place to get the colors https://convertingcolors.com/.

Now the basic idea is to guarantee the used memories (e.g., `Used` in system memory track, `Process RssAnon` in the cgroup memory track) are visualized in different red color; the unused memories (e.g., `Uused` in system memory track and the cgroup memory track) are visualized in different green colors; and the cached memories (e.g., `Buffers / Cached` in system memory track and `CGroup Mapped Files`) are visualized in different yellow colors.

Reply:
I'm not sure I understand. That just looks like a website to convert between color spaces. How were the specific shades of these colors chosen?

Reply:
I changed the comments to explain how the colors are chosen. Sorry for the previous unclear comments🙈